portfolio/src/main/webapp/index.html

```diff
@@ -55,12 +55,14 @@ <h2 align="center">Let's keep in touch!</h2>
 
       <!–– Comments ––>
       <h2 align="center">Comments</h2>
-      <p align="center"> Feel free to leave a comment down below! 💬</p>
+      <p align="center"> Feel free to leave a comment down below! 💬 Each comment has a score that
+        corresponds to the vibe your comment emits (hopefully they are good vibes 😎).</p>
+      <p align="center">(Positive score = 😄 | Negative score = ☹️)</p>
```

Comment:
suggestion: what if you emit an emoji based on the score? E.g. -1.0 to -0.8 could be 😞 and so forth

Reply:
I'll try this out!